### ISSUE TITLE
Add support for Everything v1.5a

### ISSUE THREAD

**Issue by kcorax** [Flow-Launcher/Flow.Launcher#1716]
Hello, I'm using [Everthing 1.5a]( https://www.voidtools.com/forum/viewtopic.php?t=9787) . It seems that this plugin can't detect it and gives a "Warning: Everything service is not running". Clicking on it doesn't seem to solve the issue and I get no feedback from doing that. 
Initially I assumed that the problem was the service name, but following @void's instructions to [disable the alpha version's instance naming ](https://www.voidtools.com/forum/viewtopic.php?f=12&t=9799#isntance) didn't solve the issue. Unfortunately I didn't have time to check if that indeed changed the name back to 1.4's.

Although this is a niche problem, I wonder if it makes sense to provision for cases like that on the plugin's side, for futureproofing as well. 

At any rate, thank you for your sharing your code.

**Comment by FeralFlora** [comment]
> Disable the 1.5a Instance
> 
> To use the Everything 1.4 SDK you will need to disable the "1.5a" instance. This 1.5a instance will be removed when Everything 1.5 is ready for beta/release. Completely exit out of Everything (right click the Everything system tray icon and click Exit) Open your Everything-1.5a.ini file in the same location as your Everything64.exe Add the following line to the end of the file: alpha_instance=0 Save changes and restart Everything. Everything will no longer use an instance name for window classes (IPC) Everything will continue to use the 1.5a instance name for settings, data and the Everything Service.
> 
> Have you tried this?

Thanks for this. In my case, I also had to change set this in the user settings file under the hidden AppData folder.

**Comment by tristansly** [comment]
I'm having this issue as well, I've tried many different things, don't want to have to have both 1.4 and 1.5 + a service for each running at all times (anymore), but I can't finger it out.
tried:
adding alpha_instance=0 to end of ini file
tried removing run as admin
tried enabling run as admin
same with indexing service
tried changing it in options menu alpha_instance=false
no chaching. Many thanks!!!

**Comment by ttrafford7** [comment]
you can set alpha_instance to false in the advanced options tab of Evergthing's UI

**Comment by ParhamCreate** [comment]
I already had the latest version of Everything installed on my computer, but I was still experiencing this issue. However, after uninstalling and reinstalling Everything, I checked again, and the error was resolved. It is now working correctly on my system."

**Comment by leftclot** [comment]
> you can set alpha_instance to false in the advanced options tab of Evergthing's UI

this was the easiest fix. Editing the ini file was blocked for me.

**Comment by oOtroyOo** [comment]
Same issue , even after modifying the ini file.

I noticed that the Everything Toolbar supports 1.5a. Could you possibly implement a similar approach?
see https://github.com/srwi/EverythingToolbar?tab=readme-ov-file#installation

**Comment by ttrafford7** [comment]
> Same issue , even after modifying the ini file.
> 
> I noticed that the Everything Toolbar supports 1.5a. Could you possibly implement a similar approach? see https://github.com/srwi/EverythingToolbar?tab=readme-ov-file#installation

just disable alpha instance in everything options

**Comment by oOtroyOo** [comment]
> 
> just disable alpha instance in everything options

I was using Everything-1.5.0.1391a, but it didn't work. After downgrading to Everything-1.5.0.1383a, it's functioning correctly now.

**Comment by oeloo** [comment]
> > Same issue , even after modifying the ini file.
> > I noticed that the Everything Toolbar supports 1.5a. Could you possibly implement a similar approach? see https://github.com/srwi/EverythingToolbar?tab=readme-ov-file#installation
> 
> just disable alpha instance in everything options

I have disabled alpha instance in Everything options :
![Image](https://github.com/user-attachments/assets/f099d086-d62e-4742-8c76-19ae41f4a611)
but Flow Launcher is still not using Everything Search:
![Image](https://github.com/user-attachments/assets/f440900b-1d9b-4153-baad-678b3043a540)

I am using Everything-1.5.0.1391a
Has someone been able to use Everything-1.5.0.1391a in Flow Launcher (v1.19.5)

**Comment by TheDataSeneschal** [comment]
> For info, I have Flow Launcher (v1.19.5) and using Everything-1.5.0.1391a ("C:\Program Files\Everything 1.5a\Everything64.exe") from it works fine. I do not have Everything v1.4 installed in parallel for info.

Adding the double quotes to the path thereby changing from `C:\Program Files\Everything 1.5a\Everything64.exe` to `"C:\Program Files\Everything 1.5a\Everything64.exe"` resolved my issue.

**Comment by leaumar** [comment]
Using Everything 1.5.0.1395a and Flow 1.20.1, and having done the `alpha_instance=0` and Path: `"C:\Program Files\Everything 1.5a\Everything64.exe"` config, followed by restarting Everything and Flow, it still says the Everything service isn't running (which is false, service is enabled and running) and trying to search shows the gui warning item. Is there anything else to check/try?

Edit: downgrading to 1.5.0.1391a or 1.5.0.1383a also isn't enough yet, there must be something else

Edit: got it. I had old instructions about setting `alpha_instance=0` in the program files .ini file, but the current instructions are to use the gui's advanced settings, and that actually sets it in appdata/roaming instead. _Now_ it works. TLDR: don't edit any ini files, just use the alpha's gui to disable `alpha_instance`

**Comment by Darkluna999** [comment]
> Using Everything 1.5.0.1395a and Flow 1.20.1, and having done the `alpha_instance=0` and Path: `"C:\Program Files\Everything 1.5a\Everything64.exe"` config, followed by restarting Everything and Flow, it still says the Everything service isn't running (which is false, service is enabled and running) and trying to search shows the gui warning item. Is there anything else to check/try?
> 
> Edit: downgrading to 1.5.0.1391a or 1.5.0.1383a also isn't enough yet, there must be something else
> 
> Edit: got it. I had old instructions about setting `alpha_instance=0` in the program files .ini file, but the current instructions are to use the gui's advanced settings, and that actually sets it in appdata/roaming instead. _Now_ it works. TLDR: don't edit any ini files, just use the alpha's gui to disable `alpha_instance`

The best solution！

**Comment by V7MoVknqxavTR** [comment]
It's been several years now and this is still a problem with the latest FlowLauncher build. The alpha_instance=0 fix worked for me... perhaps there can be some implementation out of the box to fix this so users aren't confused.

**Comment by deefrawley** [comment]
> It's been several years now and this is still a problem with the latest FlowLauncher build. The alpha_instance=0 fix worked for me... perhaps there can be some implementation out of the box to fix this so users aren't confused.

Rather than other applications having to have the burden of implementing workarounds for alpha builds, your efforts would better be directed at Voidtools to get them to release 1.5 as the stable branch so it works out of the box with everything else.

**Comment by Proaxel** [comment]
Thanks for the workaround y'all. To anyone else that finds this, here's a picture of the setting that you need to turn to false.
<img width="578" height="559" alt="Image" src="https://github.com/user-attachments/assets/9609b14e-28f1-4201-b6fa-70b10673e669" />

**Comment by DavidGBrett** [comment]
Proper support for this and v1.5b has just been merged into the dev branch via #4370 
It will take a while for the build to process but a new pre-release including this will show up soon here:
https://github.com/Flow-Launcher/Prereleases/releases
